libcst/_nodes/tests/test_subscript.py

```diff
@@ -17,18 +17,25 @@ class SubscriptTest(CSTNodeTest):
         (
             # Simple subscript expression
             (
-                cst.Subscript(cst.Name("foo"), cst.Index(cst.Integer("5"))),
+                cst.Subscript(
+                    cst.Name("foo"),
+                    (cst.SubscriptElement(cst.Index(cst.Integer("5"))),),
+                ),
                 "foo[5]",
                 True,
             ),
             # Test creation of subscript with slice/extslice.
             (
                 cst.Subscript(
                     cst.Name("foo"),
-                    cst.Slice(
-                        lower=cst.Integer("1"),
-                        upper=cst.Integer("2"),
-                        step=cst.Integer("3"),
+                    (
+                        cst.SubscriptElement(
+                            cst.Slice(
+                                lower=cst.Integer("1"),
+                                upper=cst.Integer("2"),
+                                step=cst.Integer("3"),
+                            )
+                        ),
```

Comment:
Just curious, do you make those changes with codemod?

Reply:
No :( Since the codemod portion of libcst isn't open-source yet, it was easier to do them by hand.